waku/v2/node/wakunode2.nim

```diff
@@ -153,6 +161,17 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   for multiaddr in announcedAddresses:
     peerInfo.addrs.add(multiaddr) # Announced addresses in index > 0
   
+  # WSS switch builder code
+  let WssSwitch = SwitchBuilder
+    .new()
+    .withAddress(hostAddress)
+    .withRng(rng)
+    .withMplex()
+    .withTransport(proc (upgr: Upgrade): Transport = WsTransport.new(upgr))
+    .withNoise()
+    .build()
+
+
   var switch = newStandardSwitch(some(nodekey), hostAddress,
```

Comment:
I understand that it's easier to test with a dedicated switch for `wss`, but I don't think we want two switches (a single `switch` supports multiple transports and multiple host endpoints already). What's more your `wssSwitch` does not have a private key, etc. My suggestion: `wakunode2` should stop relying on `newStandardSwitch()` and instead use its own constructor that is based on that function. The only difference to what was before, is that wss endpoint and transport information _can_ also be included, if enabled in config. This constructor should preferably live in another module, like `waku_setup`.